JsonLd/Response.php

```diff
@@ -18,7 +18,7 @@
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
-class JsonLdResponse extends JsonResponse
+class Response extends JsonResponse
```

Comment:
Well, I was mainly thinking of just using the `Response` object in a controller: isn't not clear as the `JsonResponse` object.
And the argument about aliased `use`s may not be relevant since a controller don't have to return at the same time HTML and JSON.